BaseTools/Source/C/Include/Arm/ProcessorBind.h

```diff
@@ -22,39 +22,96 @@
 #pragma pack()
 #endif
 
+// MU_CHANGE Starts: Fixing basic ARM definitions in Windows 

```

Comment:
Are all of these intended to be used long term? Or, are some just due to the current state of the code?

Reply:
I will try to upstream this change to edk2. My guess is that the MSC branch here was barely tested.